coalib/parsing/DefaultArgParser.py

```diff
@@ -230,6 +230,11 @@ def default_arg_parser(formatter_class=None):
         '-n', '--no-orig', const=True, action='store_const',
         help="don't create .orig backup files before patching")
 
+    misc_group.add_argument(
+        '--debug', const=True, action='store_const',
+        help='run coala in debug mode, starting ipdb on internal exceptions '
+             '(implies --verbose)')
```

Comment:
And we could use a `PipRequirement` when `--debug` flag is given to assert that `ipdb` is installed and step out and print install command if not